rotkehlchen/globaldb/handler.py

```diff
@@ -1390,6 +1380,27 @@ def get_binance_pairs(self, location: Location) -> List['BinancePair']:
                 log.debug(f'Failed to deserialize binance pair {pair}. {str(e)}')
         return pairs
 
+    def get_user_added_assets(self, user_db: 'DBHandler') -> Set[str]:
+        """
+        Create a list of the assets indentifiers added by the user AND that any user owns.
```

Comment:
```suggestion
        Create a list of the asset identifiers added by the user
```

Why also the ones that the user owns? As a user you want to export all custom assets you have input irrespective of owning it or not.